src/main/java/io/jenkins/plugins/synopsys/security/scan/input/blackduck/BlackDuck.java

```diff
@@ -1,9 +1,10 @@
 package io.jenkins.plugins.synopsys.security.scan.input.blackduck;
 
 import com.fasterxml.jackson.annotation.JsonProperty;
+import io.jenkins.plugins.synopsys.security.scan.input.AsyncMode;
 import io.jenkins.plugins.synopsys.security.scan.input.report.Reports;
 
-public class BlackDuck {
+public class BlackDuck extends AsyncMode {
```

Comment:
Why do we need to extend `AsyncMode`? We can follow similar to `Reports` and other objects. We can inject `AsyncMode` directly in `BlackDuck` class. 
If we want to extend `AsyncMode`, then we could do similar for `Reports` and other objects. But multiple inheritance will not be supported. Therefore we should follow the object injection instead of extending.